src/converters/other.ts

```diff
@@ -610,3 +608,60 @@ export function kmlCatalogItem(
 
   return { member, messages };
 }
+
+export function socrataCatalogGroup(
+  item: CatalogMember,
+  options: ConversionOptions
+): MemberResult {
+  if (!options.partial && !is.string(item.url)) {
+    return nullResult(
+      missingRequiredProp(
+        ModelType.SocrataCatalogGroup,
+        "url",
+        "string",
+        item.name
+      )
+    );
+  }
+  const member: MemberResult["member"] = {
+    type: "socrata-group",
+    name: item.name,
+  };
+
+  const propsToCopy: CopyProps[] = [
+    {
+      v7: "filterQuery",
+      v8: "filterQuery",
+      // Transform string[] to JsonObject
+      // eg ["prop=Value"] to {prop:"Value"}
+      translationFn: (f: string[]) =>
+        f.reduce<{ [key: string]: string }>((obj, curr) => {
+          const split = curr.split("=");
+          obj[split[0]] = split[1];
+          return obj;
+        }, {}),
+    },
+  ];
+
+  const propsIgnore = ["groupBy"];
```

Comment:
Are we ignoring this because we don't use it anymore in the new socrata API requests?